misc-tools/import-contest.py

```diff
@@ -0,0 +1,221 @@
+#!/usr/bin/env python3
+
+#
+# import-contest.py -- Convenience script to import a contest (including metadata, teams and
+# problems) via the command line. Supports reading credentials from ~/.netrc. See also
+# https://www.domjudge.org/docs/manual/main/import.html
```

Comment:
It would be nice to have the DOMjudge version here as this is depending on the API version

Reply:
I copied this as is. We could add the version but we will probably forget to update it…

Reply:
You should *not* put the version here literally. You could either import it from a file like `etc/config.h` (which is written from `etc/config.h.in` by configure) or create `misc-tools/import-contest.py.in` and use a substitution rule, see `misc-tools/Makefile` and the `*.in` files there.

Reply:
Ah yes that makes sense although we need something to strip the patch from if

Reply:
Any clue how to do this? We have [this](https://github.com/DOMjudge/domjudge/blob/05f7af95e4a292f91a4dba443514e29a2a00e693/doc/manual/version.py.in) in the docs, but that is not run at Makefile time. I basically want something like this but then at Make time, so we can link to `https://www.domjudge.org/docs/manual/8.1/index.html`.
Also, how do we do this for `main`? We can't link to `https://www.domjudge.org/docs/manual/8.2/index.html` since that doesn't exist.

Reply:
I'm not sure I follow: what information do you need? The domjudge version and the git branch?

That information should be collected at configure time and then when running make it would get injected into the script, see for example `misc-tools/Makefile` and the `subst-config` rule for `misc-tools/import-contest.py.in`.

Reply:
I gave it a go but as you can see I needed to modify the configure.ac and paths.mk.in file for the actual thing I needed. I'm not sure if this is what we want so feel free to modify it.

Reply:
Oh, I didn't look at the python part. I don't know whether you can create dynamic doc strings there.

But if all of this is just about getting that doc string to point to the exact URL, then maybe this is not worth it?